src/configuration/customers/login-as-customer.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Login as Customer
+---
+
+There are times when customers need help with their order. To see what they see, store administrators can "Login as Customer," which allows them to see what the customer sees and take actions on their behalf.
+
+Any actions taken while logged in "as Customer" will be applied to the actual customer's account.
+
+## To enable Login as Customer:
+
+Login as Customer is enabled in the Access Control List.
+
+1. Go to **System** > Permissions > **User Roles**.
+1. Select a Role to edit.
+1. Select Role Resources > Role Resouces. Change to 'Custom'.
+1. Under 'Customer', there are options for:
+   *  Login as Customer
+      *  Allow Login as Customer - This allows the admin user to use the Login as Customer feature.
+      *  View Login as Customer Log - Allows the admin user to see the Login as Customer Log.
+
+Then, go to **Stores** > **Configuration** > **Customers** > **Login as Customer**.
+
+| Option | Values | Description |
+|-- | -- | -- |
+| Enable Login as Customer | Yes/No | Enable/Disable the Login as Customer feature |
+| Disable Page Cache for Admin User | Yes/no | If Yes, the Page Cache will be disabled for the admin user. |
+| Store View to Log in | Automatic/Manual | Automatic will use the default store setting for that company user. Use "Manual Choose" if want the admin user to select the store view when logging in as Customer. |
+
+Disabling the page cache will ensure that the user logging in as Customer will get fresh, uncached data.
+
+Once enabled, the Login as Customer button will appear in the:
+
+*  [Customer Edit page](https://docs.magento.com/m2/ce/user_guide/customers/customer-account-update.html)
+*  [Order View page](https://docs.magento.com/m2/ce/user_guide/sales/order-processing.html#view-an-order)
+*  [Invoice View page](https://docs.magento.com/m2/ce/user_guide/sales/invoice-create.html)
+*  [Shipment View page](https://docs.magento.com/m2/ce/user_guide/sales/shipments-create.html)
+*  [Credit Memo View page](https://docs.magento.com/m2/ce/user_guide/sales/credit-memo-create.html)
+
+## Login as Customer Log
+
+Magento provides a log for the Login as Customer feature. It lists all sessions where a store administrator used the Login as Customer feature. It can be found at XXXXXXXX.
+
+<!-- Enterprise Edition -->
+
+Magento provides a log for Login as Customer actions. In Magento Commerce, actions taken under Login as Customer will be listed in the main Magento log.
+
+<!-- /Enterprise Edition -->
+
+## Permissions
```

Comment:
This section seems redundant. You already outlined the permissions above. I would suggest adding sentence or two in the following "Using Login as Customer" section that the permissions described in the earlier section are required to access these features. I would not call them "Access Control options" .

Reply:
I split as two topics - concept procedure in main customers section and config ref.